pkgs/servers/x11/xorg/default.nix

```diff
@@ -140,10 +140,10 @@ let
   }) // {inherit ;};
 
   encodings = (mkDerivation "encodings" {
-    name = "encodings-1.0.4";
```

Comment:
You should update your script to handle this case here..

It updated all 1.0.4 versions in this file to 1.0.5 versions, which may not exist and even if they did, the hash isn't changed, so it would probably still build..

Reply:
Yes. I fixed this issue recently by making it give up when a file has multiple fetchers in it. That will have to do until I figured out how to make it smarter.